stages/apps/lead/harbor.tf

```diff
@@ -1,15 +1,24 @@
+locals {
+  ingress_annotations = {
+    "nginx.ingress.kubernetes.io/force-ssl-redirect" : true
+    "nginx.ingress.kubernetes.io/proxy-body-size" : "0"
+    "kubernetes.io/ingress.class" : "toolchain-nginx"
+  }
+}
+
 data "vault_generic_secret" "harbor" {
   path = "lead/aws/${data.aws_caller_identity.current.account_id}/harbor"
 }
 
 module "harbor" {
   source = "../../../modules/tools/harbor"
 
-  enable                       = var.enable_harbor
-  cluster                      = var.cluster_name
+  count                        = var.enable_harbor ? 1 : 0
```

Comment:
this is a good change, but any outputs for this stage that used to reference `module.harbor` now need to reference `module.harbor[0]`.

Reply:
good catch, resolving this in the outputs